saas_app/saas_app/settings.py

```diff
@@ -78,9 +84,14 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#databases
 
 DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'saas',
```

Comment:
Could we populate these from the env variables set here? https://github.com/cds-snc/saas-procurement/pull/5/files#diff-67a4805fdcc6145d7b3ada2a6099a9b2e91c9d0fd108c22f95d2f01d219793d1R21-R23

like `env. POSTGRES_DB` or some other dictionary?